migrations/versions/0377_populate_org_brand_pools.py

```diff
@@ -0,0 +1,33 @@
+"""
+
+Revision ID: 0377_populate_org_brand_pools
+Revises: 0376_email_branding_pools
+Create Date: 2022-09-16 17:11:24.118619
+
+"""
+from alembic import op
+import textwrap
+
+revision = '0377_populate_org_brand_pools'
+down_revision = '0376_email_branding_pools'
+
+
+def upgrade():
+    op.execute(
+        textwrap.dedent(
+            """
+            INSERT INTO email_branding_to_organisation
+            (organisation_id, email_branding_id)
+            (SELECT organisation_id, email_branding_id FROM services
+            JOIN service_email_branding
+            ON services.id = service_id
+            WHERE email_branding_id IS NOT NULL
```

Comment:
I think we also need to check `organisation_id IS NOT NULL` - unless you're sure that there are no services without an organisation that have a branding in prod?

I was able to make one locally and it would break the migration with eg:

```
ERROR:  null value in column "organisation_id" violates not-null constraint
DETAIL:  Failing row contains (null, e3957ce2-ef36-4b21-b318-a817d439f852).
```

Reply:
Ohhh good catch, thank you!

Reply:
Added now 👍🏼